COLLABORATOR_GUIDE.md

```diff
@@ -362,21 +356,13 @@ relies on the commit being available on GitHub.
 git push --tags origin main
 ```
 
-Finally, run `npm publish` with an appropriate tag. Don't forget to supply your token.
-
-```sh
-VJS_GITHUB_USER=gkatsev VJS_GITHUB_TOKEN=my-personal-access-token npm publish --tag next
-```
-
-After it's done, verify that the GitHub release has the correct changelog output.
-This is to make sure that the CHANGELOG didn't get garbled and isn't missing pieces.
+After the tag was pushed, GitHub actions will trigger the `release` workflow, which will do the following:
 
-If the GitHub release did not work correctly, such as if the GitHub token was not provided,
-you can run it manually:
+* Publish to npm with `next` or `next-{n}` depending on your current major version.
+* Create GitHub pre-release with changelog and Netlify preview.
```

Comment:
It'll also automatically create a discussion linked to the GitHub release.

Reply:
added